gertie/Sources/Gertie/Time.swift

```diff
@@ -0,0 +1,27 @@
+/// A wall-clock time not associated with a particular date or time zone
+public struct PlainTime {
+  var hour: UInt8
+  var minute: UInt8
+
+  public init(hour: UInt8, minute: UInt8) {
+    self.hour = hour
+    self.minute = minute
+  }
+}
+
+/// A time "window"  with a start and end not
+/// associated with a particular date or time zone
+public struct PlainTimeWindow {
```

Comment:
this is basically the swift version of what you did here: https://github.com/gertrude-app/web/pull/362/files#diff-8e182d132c0f75fa146d26876c1d7ab21036b5f9aeef79ecedd391c86baae550R18